param/_utils.py

```diff
@@ -184,7 +192,7 @@ def _validate_error_prefix(parameter, attribute=None):
 
 def _is_mutable_container(value):
     """True for mutable containers, which typically need special handling when being copied"""
-    return issubclass(type(value), (abc.MutableSequence, abc.MutableSet, abc.MutableMapping))
+    return issubclass(type(value), MUTABLE_TYPES)
```

Comment:
I am curious as to why `isinstance(value, MUTABLE_TYPES)` was not used here.

Reply:
Excellent question I don't have a good answer for.